src/blog/2023/09/modernize-your-legacy-industrial-data-part2.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Modernize your legacy industrial data. Part 2.
+subtitle: A deeper dive with into making the most of legacy industrial data from the likes of Modbus and older, non IIoT protocols and putting it to work in an IIoT world.
+description: A deeper dive with into making the most of legacy industrial data from the likes of Modbus and older, non IIoT protocols and putting it to work in an IIoT world.
+date: 2023-09-17
+authors: ["stephen-mclaughlin"]
+image: "/blog/2023/09/images/industrial-legacy-data-blog-image.png"
+tags:
+    - posts
+    - node-red
+    - community
+---
+
+In an article published recently, I introduced the topic of working with legacy industrial data from the likes of Modbus and older, non IIoT protocols and putting it to work in an IIoT world. 
+We looked at some of the challenges and how Node-RED with `node-red-contrib-buffer-parser` node can help. 
+
+In this article, I will dive a little deeper into the topic and discuss some of the finer details. I hope to demonstrate a smarter approach that can make a huge difference to data accuracy, performance and maintainability while significantly reducing developer time. Not only that, ending with a no-code solution.
+
+See the [previous article](modernize-your-legacy-industrial-data-part2.md) for the introduction and overview.
+
+<!--more-->
+
+### In the beginning...
+
+In order to generate IIoT data from legacy systems we first need to _grab_ that data.
```

Comment:
```suggestion
In order to convert the data from legacy systems to IIoT native data we first need to _grab_ that data.
```

Reply:
Applied change _similar_ to suggestion in [38834ba](https://github.com/flowforge/website/pull/1123/commits/38834bab3664f688aade56cdaebbe0c7175a7d9d)